File: backend/models/activity.py
from sqlalchemy import ForeignKey, Table, Column
from sqlalchemy.sql.sqltypes import Integer, String, Float, Boolean, Date
from config.db import meta, engine

activities = Table("activities", meta, 
    Column("id", Integer, primary_key=True),
    Column("type", Integer, ForeignKey("activityTypes.id")),
    Column("date", Date),
    Column("name", String(255)),
    Column("hours", Integer),
    Column("price", Integer),
    Column("iva", Integer),
    Column("invoice", Boolean),
    Column("getPaid", Boolean),
    Column("notes", String(355)))

meta.create_all(engine)

File: frontend/lib/models/activity.dart
import 'package:frontend/models/activityType.dart';
import 'package:frontend/models/company.dart';

class Activity {
  int id;
  ActivityType type;
  DateTime activityDate;
  String name;
  Company company;
  int hours;
  int price;
  int iva;
  bool invoice;
  bool getPaid;
  String notes;

  Activity(
      this.id, this.type, this.activityDate, this.name, this.company, this.hours, this.price, this.iva, this.invoice, this.getPaid, this.notes);
}
